PC/_wmimodule.cpp

```diff
@@ -279,9 +279,11 @@ _wmi_exec_query_impl(PyObject *module, PyObject *query)
     // a timeout.  The initEvent will be set after COM initialization, it will
     // take a longer time when first initialized.  The connectEvent will be set
     // after connected to WMI.
-    err = wait_event(data.initEvent, 1000);
     if (!err) {
-        err = wait_event(data.connectEvent, 100);
+        err = wait_event(data.initEvent, 1000);
+        if (!err) {
+            err = wait_event(data.connectEvent, 100);
+        }
```

Comment:
We can reduce the nestinglike this:

```suggestion
        err = wait_event(data.initEvent, 1000);
    }
    if (!err) {
        err = wait_event(data.connectEvent, 100);
```

Reply:
Normally I'm a fan of reducing nesting, but I think in this case it better reflects that the two events are connected and if we spread it out it looks like they're unrelated.